kovri.i2p/_i18n/es.yml

```diff
@@ -12,17 +12,17 @@ langs:
   de: Deutsch
   da: Dansk
 titles:
-  index: ¿Que es Kovri?
-  get-started: Get Started
-  intro: Introduction
+  index: ¿Qué es Kovri?
+  get-started: Tareas iniciales
+  intro: Introducción
   acknowledgements: Agradecimientos
-  quality: Calidad
-  contributing: Pautas Generales para contribuir
-  building: Compilando.
-  faq: FAQ
-  style: Guia de estilo
-  user: Guia de usuario
+  quality: Seguros de calidad
+  contributing: Guía para contribuir
```

Comment:
I think "Pautas Generales"is better, you can check how Mozilla did it here: https://www.mozilla.org/es-ES/about/governance/policies/participation/ change to spanish at the bottom.

Reply:
It could very well be the better solution.  The reason I changed it was for it to be the same as the title of a link in some of the files.  Feel free to change it back and make sure the link is updated